apps/devhub/views.py

```diff
@@ -831,6 +831,12 @@ def json_upload_detail(request, upload, addon_slug=None):
     if result['validation']:
         try:
             pkg = parse_addon(upload, addon=addon)
+            if (pkg.get('is_experiment', False) and
+                    not acl.action_allowed(request, 'Experiments', 'submit')):
```

Comment:
Can you move this into `parse_addon`? This code path isn't hit in every case where you can create an add-on but `parse_addon` should always be run. If we land this then I have a feeling anyone will be able to create an experiment through the signing API.